src/ServiceInsight/Saga/SagaWindowViewModel.cs

```diff
@@ -99,6 +106,11 @@ string GetMajorVersion(StoredMessage message)
             return version?.Split('.').First();
         }
 
+        void ClearSaga()
+        {
+            Data = null;
+        }
+
```

Comment:
Is it worth creating this separate method instead of inlining in `Handle(ServiceControlConnectionChanged)`?